src/app/page.js

```diff
@@ -2,14 +2,19 @@
 
 import Categories from "@/components/Categories"
 import FirstBanner from "@/components/FirstBanner"
+import Flex from "@/components/Flex"
 import MostPopular from "@/components/MostPopular"
+import RecentPosts from "@/components/RecentPosts"
 
 export default function Home() {
   return (
     <>
       <FirstBanner />
       <Categories />
-      <MostPopular />
+      <Flex gap={"20px"} alignItems={"flex-start"}>
```

Comment:
for props containing strings, you can pass "" directly. Like: `<Flex gap="20px" .../>`